test/scripts/agent-images/Containerfile-e2e-base.local

```
@@ -17,7 +17,11 @@ COPY bin/rpm/flightctl-agent-*.rpm /tmp/
 RUN dnf install -y /tmp/flightctl-agent-*.rpm && \
     systemctl enable flightctl-agent.service
 
-RUN dnf install -y greenboot greenboot-default-health-checks
+RUN dnf -y update && \
+    dnf -y install greenboot greenboot-default-health-checks podman python3 python3-pip && \
+    dnf clean all
+
+RUN pip3 install podman-compose
```

Let's use epel instead of pip:

```suggestion
RUN dnf install -y epel-release epel-next-release
RUN dnf install -y greenboot greenboot-default-health-checks podman-compose
```

---

thx looks better updated